packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts

```diff
@@ -126,8 +127,9 @@ export async function createDeclarationCreator(
         for (let dep of deps) {
           let sourceFile = program!.getSourceFile(dep);
           if (!sourceFile) {
-            throw new Error(
-              "This is an internal error, please open an issue if you see this: source file not found"
+            throw new FatalError(
+              `Could not generate type declarations because ${dep} does not exist or is not a TypeScript file`,
+              dep
```

Comment:
I wasn't sure what to have as the scope. Let me know if it should be something else

Reply:
It should be `pkg.name` so `createDeclarationCreator` should be changed to accept `Package`

Reply:
@mitchellhamilton FYI I passed in a new arg instead of all of `Package`